client/dashboard/sites/settings-transfer-site/start-site-transfer-form.tsx

```diff
@@ -0,0 +1,214 @@
+				</Text>
+			</VStack>
+			<VStack spacing={ 5 } style={ { padding: '8px 0' } }>
+				<Notice variant="warning" density="medium">
+					{ __( 'Please read the following carefully. Transferring a site cannot be undone.' ) }
+				</Notice>
+				<VStack spacing={ 6 } style={ { padding: '8px 0' } }>
+					<List title={ __( 'Content and ownership' ) }>
+						<li>
+							{ createInterpolateElement(
+								__(
+									'You’ll be removed as owner of <siteSlug /> and <newOwnerEmail /> will the new owner from now on.'
+								),
+								{
+									siteSlug: renderSiteSlug(),
+									newOwnerEmail: renderNewOwnerEmail(),
+								}
+							) }
+						</li>
+						<li>
+							{ createInterpolateElement(
+								__( 'You will keep your admin access unless <newOwnerEmail /> removes you.' ),
+								{
+									newOwnerEmail: renderNewOwnerEmail(),
+								}
+							) }
+						</li>
+						<li>
+							{ createInterpolateElement(
+								__( 'Your posts on <siteSlug /> will remain authored by your account.' ),
+								{
+									siteSlug: renderSiteSlug(),
+								}
+							) }
+						</li>
+						{ site.is_wpcom_atomic && ! site.is_wpcom_staging_site && (
+							<li>
+								{ createInterpolateElement(
+									__(
+										'If your site <siteSlug /> has a staging site, it will be transferred to <newOwnerEmail />.'
+									),
+									{
+										siteSlug: renderSiteSlug(),
+										newOwnerEmail: renderNewOwnerEmail(),
+									}
+								) }
+							</li>
+						) }
+					</List>
+					{ /* TODO: Check there is any active purchase on current user. */ }
+					<List title={ __( 'Upgrades' ) }>
+						<li>
+							{ createInterpolateElement(
+								__(
+									'Your paid upgrades on <siteSlug /> will be transferred to <newOwnerEmail /> and will remain with the site.'
+								),
+								{
+									siteSlug: renderSiteSlug(),
+									newOwnerEmail: renderNewOwnerEmail(),
+								}
+							) }
+						</li>
+					</List>
+					<List title={ __( 'Domains' ) }>
+						<li>
+							{ createInterpolateElement(
+								__(
+									'The domain name <siteSlug /> will be transferred to <newOwnerEmail /> and will remain working on the site.'
+								),
+								{
+									siteSlug: renderSiteSlug(),
+									newOwnerEmail: renderNewOwnerEmail(),
+								}
+							) }
+						</li>
+					</List>
+				</VStack>
+				<form onSubmit={ handleSubmit }>
+					<VStack>
+						<span style={ { textTransform: 'uppercase', fontSize: '11px', fontWeight: 500 } }>
+							{ __( 'To transfer your site, review and accept the following statements:' ) }
+						</span>
+						<DataForm< StartSiteTransferFormData >
+							data={ formData }
+							fields={ fields }
+							form={ form }
+							onChange={ ( edits: Partial< StartSiteTransferFormData > ) => {
+								setFormData( ( data ) => ( { ...data, ...edits } ) );
+							} }
+						/>
+						<HStack justify="flex-start" style={ { paddingTop: '8px' } }>
```

Comment:
Not a blocker, but I find it weird that we are still using `paddingTop` with `<VStack>`; seems to defeat the purpose 🤔

Reply:
yeah...but the gap is different and I don't want to have another VStack just for the padding 😞